_includes/pbc/all/install-features/202307.0/install-the-product-rating-and-reviews-feature.md

```diff
@@ -1,201 +1,1212 @@
-    use Spryker\Zed\ProductReviewCollector\Communication\Plugin\ProductReviewCollectorSearchPlugin;
-    use Spryker\Zed\ProductReviewCollector\Communication\Plugin\ProductAbstractReviewCollectorStoragePlugin;
-    // ...
+class SearchElasticsearchConfig extends SprykerSearchElasticsearchConfig
+{
+    protected const SUPPORTED_SOURCE_IDENTIFIERS = [
+        'product-review',
+    ];
+}
+```
+
+2. Set up a new source for Product Reviews:
+
+```bash
+console search:setup:source-map
+```
 
-    class CollectorDependencyProvider extends SprykerCollectorDependencyProvider
+3. In `\Pyz\Zed\StoreStorage\StoreStorageConfig`, adjust the `StoreStorage` module's configuration:
+
+**src/Pyz/Zed/StoreStorage/StoreStorageConfig.php**
+
+```php
+<?php
+
+namespace Pyz\Zed\StoreStorage;
+
+use Spryker\Shared\ProductReviewSearch\ProductReviewSearchConfig;
+use Spryker\Zed\StoreStorage\StoreStorageConfig as SprykerStoreStorageConfig;
+
+class StoreStorageConfig extends SprykerStoreStorageConfig
+{
+    /**
+     * @return array<string>
+     */
+    public function getStoreCreationResourcesToReSync(): array
     {
-        /**
-         * @param \Spryker\Zed\Kernel\Container $container
-         *
-         * @return \Spryker\Zed\Kernel\Container
-         */
-        public function provideBusinessLayerDependencies(Container $container)
-        {
-            // ...
+        return [
+            ProductReviewSearchConfig::PRODUCT_REVIEW_RESOURCE_NAME,
+        ];
+    }
+}
+```
 
-            $container[static::SEARCH_PLUGINS] = function (Container $container) {
-                return [
-                    // ...
-                    ProductReviewConfig::RESOURCE_TYPE_PRODUCT_REVIEW => new ProductReviewCollectorSearchPlugin(),
-                ];
-            };
+4. Configure the synchronization pool and event queue name:
 
-            $container[static::STORAGE_PLUGINS] = function (Container $container) {
-                return [
-                    // ...
-                    ProductReviewConfig::RESOURCE_TYPE_PRODUCT_ABSTRACT_REVIEW => new ProductAbstractReviewCollectorStoragePlugin(),
-                ];
-            };
+**src/Pyz/Zed/ProductReviewSearch/ProductReviewSearchConfig.php**
 
+```php
+<?php
 
-            // ...
-        }
+namespace Pyz\Zed\ProductReviewSearch;
+
+use Pyz\Zed\Synchronization\SynchronizationConfig;
+use Spryker\Shared\Publisher\PublisherConfig;
+use Spryker\Zed\ProductReviewSearch\ProductReviewSearchConfig as SprykerProductReviewSearchConfig;
+
+class ProductReviewSearchConfig extends SprykerProductReviewSearchConfig
+{
+    /**
+     * @return string|null
+     */
+    public function getProductReviewSynchronizationPoolName(): ?string
+    {
+        return SynchronizationConfig::DEFAULT_SYNCHRONIZATION_POOL_NAME;
+    }
+
+    /**
+     * @return string|null
+     */
+    public function getEventQueueName(): ?string
+    {
+        return PublisherConfig::PUBLISH_QUEUE;
     }
+}
 ```
 
+5. Set up resync feature:
```

Comment:
do you mean this?
```suggestion
5. Set up and resync the feature:
```

Reply:
You are right. Fixed to 'Add synchronization plugins:' (we have this wording everywhere)